### DISCUSSION TITLE
学而思网页直播

### DISCUSSION THREAD

**Discussion by yyscream** [xifangczy/cat-catch#102] [Q&A]
好像只能抓取到摄像头无法抓取到视频链接如下https://xueyan.xueersi.com/web-live/index.html?bizId=3&stuCouId=141998697&classId=3690698&fromType=5&planId=1088381&forceweb=1&planVersion=28
有没有好心人帮忙看一下

**Comment by xifangczy** [comment]
需要账号 没办法帮你看

**Reply by yyscream** [reply]
我提供账号能否帮我看一下

**Reply by xifangczy** [reply]
https://o2bmm.gitbook.io/cat-catch/issues 这有邮箱。你发邮件吧
